src/LuauAST/types/nodes.ts

```diff
@@ -112,6 +112,10 @@ export interface IfExpression extends luau.BaseExpression<luau.SyntaxKind.IfExpr
 	alternative: luau.Expression;
 }
 
+export interface InterpolatedString extends luau.BaseExpression<luau.SyntaxKind.InterpolatedString> {
+	segments: luau.List<luau.Expression>;
```

Comment:
bikeshedding: does the name `segments` need to be this long, or could something simpler like `parts` fit?